data-types/collections/hashes.rb

```diff
@@ -1,3 +1,17 @@
+#hashes.rb contains a series of methods that need to be completed by you.
```

Comment:
convention is for a space between the `#` and the beginning of the text:

```ruby
#not formatted correctly
# formatted correctly
```

Reply:
awesome.  I'll do that from here on out.